README.md

```diff
@@ -40,21 +47,22 @@ import (
     "github.com/go-flac/go-flac"
 )
 
-func extractFLACComment(fileName string) *flacvorbis.MetadataBlockVorbisComment {
+func addFLACTitle(fileName string, title []byte) {
 	f, err := flac.ParseFile(fileName)
 	if err != nil {
 		panic(err)
 	}
-    
-    var cmt *flacvorbis.MetadataBlockVorbisComment
-	for _, meta := range f.Meta {
-		if meta.Type == flac.VorbisComment {
-			cmt, err = flacvorbis.ParseFromMetaDataBlock(*meta)
-			if err != nil {
-				panic(err)
-			}
-		}
-    }
-    return pic
+	cmts, idx := extractFLACComment(f)
+	if cmts == nil && idx > 0 {
+		cmts := flacvorbis.New()
```

Comment:
Use `=` here, otherwise another scoped variable will be created and `cmts` will still be nil outside of the if statement

Reply:
True that. Updated accordingly